Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/SN3A-Memory-Consumption/memory-consumption

```diff
@@ -0,0 +1,177 @@
+
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id SN3A 3-functional "Check if the memory consumption doesn't cross the desired threshold for known workload" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+echo "********Deprovisioning Busybox Application*******"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=busybox-deprovision-mem-check metadata="")
+echo $test_name
+cd litmus
+
+################
+# LitmusBook 3 #
+################
+
+: << EOF
+  --------------------------------------------------------------------------------------------------------------------     
+ | specAttribute     | kind   |         default value               | test specifc value                              |
+  ------------------------------------------------------------------------------------------------------------------- |
+ | appLabel          | env    | app=busybox-sts                     | app=busybox-mem-jiva                            |
+ | pvcName           | env    | openebs-busybox                     | busybox-ctrl-kill                               | 
+ | appNamespace      | env    | app-busybox-ns                      | mem-check-jiva                                  | 
+ | storage clas      | env    | openebs-cstor-sparse                | openebs-jiva-default                            |
+ | deploy type       | env    | statefulset                         | deployment                                      |
+ | Action            | env    | value: provision                    | value: deprovision                              |
+ | Litmus Job name   | name   | generateName: litmus-busybox-deploy | generateName: busybox-mem-check-deprovision-jiva|
+ | Litmus job label  | label  | app: busybox-litmus                 | busybox-deprovision-mem-check-jiva              |
+  ---------------------------------------------------------------------------------------------------------------------
+EOF
+
+cp apps/busybox/deployers/run_litmus_test.yml deprovision_mem_check_jiva.yml
+sed -i -e 's/generateName: litmus-busybox-deploy/generateName: busybox-mem-check-deprovision-jiva/g' \ 
+-e 's/app: busybox-litmus/app: busybox-deprovision-mem-check-jiva/g' \
+-e 's/app=busybox-sts/app=busybox-mem-jiva/g' \
+-e 's/value: statefulset/value: deployment/g' \
+-e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' \
+-e 's/value: app-busybox-ns/value: mem-check-jiva/g' \
+-e 's/value: provision/value: deprovision/g' deprovision_mem_check_jiva.yml
+
+echo "Running the litmus test for Busybox Deployment Scaleup.."
+cat deprovision_mem_check_jiva.yml
+
+bash ../Openshift-EE/utils/litmus_job_runner label='app:busybox-deprovision-mem-check-jiva' job=deprovision_mem_check.yml
+cd ..
+bash Openshift-EE/utils/dump_cluster_state;
+bash Openshift-EE/utils/event_updater jobname:memory-consumption $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
```

Comment:
@harshita-sharma-mayadata check the job_name is this `memory-consumption` or memory-consumption-jiva`